### PR TITLE
Improve Documentation

### PR DESCRIPTION
**Pull request by Veerhan-glitch**
This is with regards to issue #17 

What has been done:

Updated README.md and src/main.rs
Created CONTRIBUTING.md and CODE_OF_CONDUCT.md

What hasn't been done:

contents of CODE_OF_CONDUCT.md have been left blank, (I cant do this, as this is something specific to the repo mantaner)

Changes in README.md:
1) Added a table of contents
2) some minor changes here and there

If you need me to change/improve on anything please tell me

regards

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+1. Rmember to read [code of conduct](./code_of_conduct).
+
+### For PR
+
+1. Fork the repository to your own GitHub account.
+2. Clone your forked repository to your local machine.
+3. Create a new branch for your work with a proper well defined name.
+4. Make your changes and commit them with meaningful commit messages.
+5. Open [PRs](https://github.com/euanwm/OpenWeightlifting/pulls) to development branch (Remember to [Squash](https://docs.github.com/en/desktop/managing-commits/squashing-commits-in-github-desktop) before making a pr).
```
Comment (arncv) [inline CONTRIBUTING.md]: Awesome work! Could you change this link- https://github.com/euanwm/OpenWeightlifting/pulls to https://github.com/arncv/TextScribe/pulls

**Comment by arncv** [comment]
Changed this link- https://github.com/euanwm/OpenWeightlifting/pulls to https://github.com/arncv/TextScribe/pulls

File: CONTRIBUTING.md (hunk 1 above)
Comment (arncv) [reply CONTRIBUTING.md]: No worries, changed it right now!

**Review by arncv — APPROVED** [review]

**Comment by arncv** [comment]
Thank you for contribution! Merging it 🚀

File: CONTRIBUTING.md (hunk 1 above)
Comment (Veerhan-glitch) [reply CONTRIBUTING.md]: oh sorry i did not see it,
Thx for taking care of it and merging my pr

**Pull request merged**